package.json

```diff
@@ -43,6 +43,7 @@
     "test": "eslint Gruntfile.js lib test/test.js && nodeunit test/test.js"
   },
   "dependencies": {
+    "stream-browserify": "^2.0.1",
```

Comment:
I don't think this is necessary. According to the [node-browserify npm page](https://www.npmjs.com/package/stream-browserify), we'll get this automatically when using browserify.